### PR TITLE
Help Page

### PR DESCRIPTION
**Pull request by NikolasGraham**
Good enough? Want anything else? Anything changed?

### CODE REVIEW

File: src/containers/__Routing/Routing.js [hunk 1]
```diff
@@ -136,6 +137,16 @@ const RoutingBody = (props) => {
     //     }
     // };
 
+    const helpPageCommand = {
+        command: ["mirror mirror help page", "mirror mirror on the wall help page", "mirror mirror go to help page", "mirror mirror take me to help page",
+                        "mirror mirror I have fallen and need a help page"],
```
Comment (JeffHowes) [inline src/containers/__Routing/Routing.js:151]: LOL!!! Thanks for the laugh, I needed it!!!

**Comment by JeffHowes** [comment]
Looks good Nik!

File: src/containers/HelpPage/HelpPage.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    return (
+        <div>
+            <h1 style={{color: "teal"}}>Primary Voice Commands</h1>
+            <Row>
+                <Col lg={5}>
+                    <h3 style={{color: "blue"}}>Page Commands</h3>
+                    <h5><label style={{color: "red", fontSize: "28px"}}>Home: </label> Mirror mirror go home</h5>
+                    <h5><label style={{color: "red", fontSize: "28px"}}>Voice Demo: </label> Mirror mirror go to voice demo page</h5>
+                    <h5><label style={{color: "red", fontSize: "28px"}}>Face Demo: </label> Mirror mirror go to face demo page</h5>
+                    <h5><label style={{color: "red", fontSize: "28px"}}>Gesture Demo 1: </label> Mirror mirror I want to paint</h5>
```
Comment (toha87) [inline src/containers/HelpPage/HelpPage.js]: You can add: mirror mirror I want to play click me game

File: src/containers/HelpPage/HelpPage.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                </Col>
+                <div>
+                    <h3 style={{color: "blue"}}>All Commands</h3>
+                    <h5 style={{color: "blue"}}>(Some commands only function in certain instances)</h5>
+                    {
+                        voiceContext.intendArray.map((command) => {
+                            return <div>
+                                {
+                                    command.command.map((command) => {
+                                        return <a style={{fontSize: "18px"}}> {JSON.stringify(command)} </a>
```
Comment (toha87) [inline src/containers/HelpPage/HelpPage.js:52]: 👍

**Review by toha87 — APPROVED** [review]

**Review by JeffHowes — APPROVED** [review]
I would recommend changing the "(Some commands only function in certain instances)" to something like "(Some voice commands only work in certain pages)".  Keeping the verbiage simple for everyone.

File: src/containers/__Routing/Routing.js (hunk 1 above)
Comment (blazinaj) [reply src/containers/__Routing/Routing.js:151]: 🤣

**Review by blazinaj — APPROVED** [review]

**Pull request merged**